pom.xml

```diff
@@ -513,6 +513,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.mojang</groupId>
+            <artifactId>authlib</artifactId>
+            <version>1.5.25</version>
```

Comment:
```suggestion
            <version>4.0.43</version>
```

Reply:
This is the version we use in dough (which we figured out is very outdated)

where does this version come from? We should update dough before this - it also is not at all a blocker for this PR

Reply:
This version comes from maven repo.
This 1.5.25 has a bunch fo security risks.
No blocker for this PR but it would be really nice to be updated.
Would it cause issues depending on a newer build in here then dough has?

Reply:
It could easily yes, having dependencies out of sync is never fun